src/views/forms/AccountManagementFormsNew/WithdrawForm/NetworkSelectMenu.tsx

```diff
@@ -0,0 +1,156 @@
+import { Chain } from '@skip-go/client';
+import tw from 'twin.macro';
+
+import { cctpTokensByChainId, isHighFeeChainId, isLowFeeChainId } from '@/constants/cctp';
+import { SUPPORTED_COSMOS_CHAINS } from '@/constants/graz';
+import { STRING_KEYS } from '@/constants/localization';
+import { TransferType } from '@/constants/transfers';
+import { ConnectorType, WalletType } from '@/constants/wallets';
+
+import { useAccounts } from '@/hooks/useAccounts';
+import { useEnvFeatures } from '@/hooks/useEnvFeatures';
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { SearchSelectMenu } from '@/components/SearchSelectMenu';
+
+import { getSelectedDydxChainId } from '@/state/appSelectors';
+import { useAppSelector } from '@/state/appTypes';
+
+import { isTruthy } from '@/lib/isTruthy';
+
+import exchanges from '../../../../../public/configs/exchanges.json';
+import { FeeLevel, FeeLevelTag } from '../FeeLevelTag';
+
+type ElementProps = {
+  selectedExchange?: string;
+  selectedChain?: string;
+  onSelect: (name: string, type: 'chain' | 'exchange') => void;
+  chains: Chain[];
+};
+
+// TODO: fix this
+// our menu item types are wrong (has label typed as a react node even though we use string)
```

Comment:
string is a valid React node!

Reply:
ooo i must have misread the error message. 🔥 thank u